dbt/adapters/redshift/relation_configs/sort.py

```diff
@@ -138,7 +139,7 @@ def parse_relation_config(cls, relation_config: RelationConfig) -> Dict[str, Any
         return config_dict
 
     @classmethod
-    def parse_relation_results(cls, relation_results_entry: "agate.Row") -> dict:
+    def parse_relation_results(cls, relation_results_entry: MappedSequence) -> dict:
```

Comment:
This type change is fine since it's almost the same thing. But we're trying to lazy load `agate` for type checking since it adds a material amount of time to execution. Since `agate` is already behind a `TYPE_CHECKING` gate on line 19, can you update this to `agate.MappedSequence` and remove the import on line 5?

Reply:
fixed